managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/EntryCacheImpl.java

```diff
@@ -234,7 +228,11 @@ private void asyncReadEntry0(ReadHandle lh, PositionImpl position, final ReadEnt
                         } finally {
                             ledgerEntries.close();
                         }
-                    }, ml.getExecutor().chooseThread(ml.getName()));
```

Comment:
What's the reason of removing this async thread executor? IMHO, it may cause some concurrent issue.

Reply:
Thank you for pointing out this problem, it may be better to use asynchronous operations.